### ISSUE TITLE
Inconsistent interaction with volumetric sensor area

### ISSUE THREAD

**Issue by JanikHelbig** [tsunamayo/Starship-EVO#2005]
### Short Description of the issue:

The volumetric sensor triggers when **any** part of the player model enters the volume, including - but not limited to - the player's tool. When the player is within a certain distance of a volume it is enough for the player to turn by moving their to move their tool inside the volume and trigger the sensor. This makes the sensor feel a bit inconsitent and uncalculable. I would expect the player have a capsule-like collider in these instances. (This probaly extends to laser sensors and gravity fields, but I didn't test those yet tbh)

I'm including 2 screenshot that show opening a door by simply turning the player and no other movement.

### Environment
- Starship EVO version: 20w19d

### Additional information:

![Closed](https://user-images.githubusercontent.com/33556800/81474797-0df86e00-9208-11ea-8983-497fd7eee5b5.png)
![Open](https://user-images.githubusercontent.com/33556800/81474799-0fc23180-9208-11ea-9dcc-4366cdccc3ec.png)

**Comment by tsunamayo** [comment]
Yes I could improve that indeed. The laser does a real raycast, the volume is less precise.
Thanks
